webpack/assets/javascripts/react_app/components/Architectures/Welcome.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl } from '../../common/helpers';
+
+export const WelcomeArchitecture = ({ canCreate }) => {
+  const action = canCreate && {
+    title: __('Create Architecture'),
+    url: foremanUrl('/architectures/new'),
+  };
+  const description = (
+    <>
+      <p>
```

Comment:
There is no need for both `<>` and `<p>` - `<>` is needed when you want to group several elements together without wrapping them in another element.